src/main/scala/cognite/spark/v1/CdfSparkAuth.scala

```diff
@@ -16,17 +17,27 @@ object CdfSparkAuth {
       IO(AuthProvider(auth))
   }
 
-  final case class OAuth2ClientCredentials(credentials: OAuth2.ClientCredentials) extends CdfSparkAuth {
+  final case class OAuth2ClientCredentials(credentials: OAuth2.ClientCredentials)(
+      implicit sttpBackend: SttpBackend[IO, Any],
```

Comment:
This doesn't seem right to me, but I don't have time to come up with a better solution at the moment. At least for the `IORuntime`, I think we only ever use the one in `object CdpConnector` so we can just import that directly.

Reply:
I removed the IORuntime and use the import directly to make it less redundant 😁